services/queue/config.yml

```diff
@@ -34,25 +34,15 @@ defaults:
     # schedulerId inside a task-group where all tasks are about to expire.
     taskGroupExpiresExtension:     6220800 # 3 days as seconds
 
-    # Prefix for azure queues (at most 6 characters)
-    queuePrefix:                  queue
-
-    # Name of azure queue for tracking claim expiration
-    claimQueue:                   claim-queue
-
-    # Name of azure queue for tracking deadlines
-    deadlineQueue:                deadline-queue
-
-    # Name of azure queue for tracking resolved tasks (dependency tracker)
-    resolvedQueue:                resolved-queue
-
     # Number of ms before deadline expiration message arrives, past deadline
     deadlineDelay:                60000
 
     # Configuration for deadline-resolver
     deadlineResolver:
       # Number of ms to sleep after polling from an empty queue
       pollingDelay:               5000
+      # Number of records to process at once
+      count:                      32
```

Comment:
this was the default from azqueue times, maybe it would be safe to increase this number.

Reply:
It's probably not worthwhile -- the deadline resolver gets this many messages and then handles them with `Promise.all`, so it's basically defining the concurrency of a single resolver instance. That concurrency is probably already limited by the maximum number of DB connections. I suspect that if the deadline resolver is falling behind, `parallelism` is already sufficient to adjust this concurrency.

Anyway, adding this as a config.yml value doesn't hurt, but I wouldn't suggest making it configurable with an env var.

Reply:
Indeed! Maybe I should put it as a constant in code instead, and document the reasoning behind this number